clippy.toml

```diff
@@ -0,0 +1 @@
+avoid-breaking-exported-api = false
```

Comment:
Not directly related to the idea of this PR but IIRC there was a way to configure clippy inside `Cargo.toml`, can we do that instead of committing this?

Reply:
Sadly I don't think so. Inside the `Cargo.toml` the lints (for all targets) can be configured. Inside `clippy.toml` the lint options can be configured.